src/main/java/VASSAL/build/module/gamepieceimage/SwatchComboBox.java

```diff
@@ -71,10 +71,10 @@ public SwatchRenderer() {
      * value and returns the label, set up to display the text and image.
      */
     @Override
-    public Component getListCellRendererComponent(JList list, Object value, int index, boolean isSelected,
+    public Component getListCellRendererComponent(JList<? extends String> list, String value, int index, boolean isSelected,
```

Comment:
The wonders of type erasure, after this change this is how the compiler writes the bytecode:
```
    public Component getListCellRendererComponent(JList list, Object value, int index, boolean isSelected, boolean cellHasFocus) {
      ColorSwatch swatch = ColorManager.getColorManager().getColorSwatch((String)value);
```
Looks exactly like the source code before the change and should be binary compatible.